web/api/v1/api.go

```diff
@@ -909,11 +911,13 @@ type alertingRule struct {
 }
 
 type recordingRule struct {
-	Name      string           `json:"name"`
-	Query     string           `json:"query"`
-	Labels    labels.Labels    `json:"labels,omitempty"`
-	Health    rules.RuleHealth `json:"health"`
-	LastError string           `json:"lastError,omitempty"`
+	Name           string           `json:"name"`
+	Query          string           `json:"query"`
+	Labels         labels.Labels    `json:"labels,omitempty"`
+	Health         rules.RuleHealth `json:"health"`
+	LastError      string           `json:"lastError"`
+	EvaluationTime int64            `json:"evaluationTime"`
```

Comment:
We typically use "...Time" for specific timestamps, not for durations, and since we want to export in seconds, we use a float64 for e.g. the target scrape duration. So better would be:

```suggestion
	EvaluationDuration float64            `json:"evaluationDuration"`
```